src/_posts/platform/deployment/2000-01-01-deploy-with-git.md

```diff
@@ -82,19 +82,38 @@ platform/app/2000-01-01-procfile %}).
 
 Once a deployment has succeeded, you can recover the code you pushed to Scalingo using:
 
-```
+```bash
 git clone --origin scalingo git@ssh.osc-fr1.scalingo.com:my-app.git
 ```
 
 This will create a copy of the Git history hosted in Scalingo repositories.
 It will also add the `scalingo` remote to simplify future pushes.
 
 {% note %}
-  Only Git operations will be available via `git clone`. If you deployed your app
-  in any other way (GitLab / GitHub integration, archive, ...) the changes made
-  to your code won't be available.
+  Only Git operations will be available via `git clone`.
+  If you deployed your app in any other way (GitLab / GitHub integration, archive, ...)
+  the changes made to your code won't be available.
```

Comment:
Should we add something like `Git is used at Scalingo as a deployment tool and should not be used as your code versioning platform.`  WDYT?

Reply:
There is already a message about that in the page

Reply:
![image](https://user-images.githubusercontent.com/24525868/218530017-f88da0ba-9eb2-44d1-8163-f6b51615af56.png)

Reply:
Ahh, you mean change the the warning. Okay for that

Reply:
The message is already displayed in the page:
![image](https://user-images.githubusercontent.com/24525868/218530369-147f2e3b-cbb6-4e2b-ac40-c99264c056ed.png)